spec/podfile/dsl_spec.rb

```diff
@@ -405,7 +405,16 @@ module Pod
           platform :ios
           podspec :path => banalib_path
         end
-        podfile.dependencies.map(&:name).should == %w(monkey)
+        podfile.dependencies.map(&:name).should == %w(monkey AFNetworking SDWebImage)
+      end
+
+      it 'it can use use the dependencies of a podspec\'s subspec' do
```

Comment:
nit: remove the extra 'it'

Reply:
😂I just copy the description from last spec ( Line 402 ). Should I remove them?

Reply:
just the the beginning so it's `it 'can use use the dependencies of a podspec\'s subspec' do`